tsconfig.json

```diff
@@ -7,12 +7,12 @@
     "moduleResolution": "node",
     "noEmit": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["dom", "es2017", "esnext.asynciterable"],
+    "lib": ["ES2020"],
     "module": "commonjs",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "es2017"
+    "target": "ES2020"
```

Comment:
According to https://node.green/#ES2020 Node.js v14.14 doesn't fully support ES2020 but we're not using `spread parameters after optional chaining` so that's fine, if we ever do use it the CI will catch it assuming v14 doesn't update to a version of v8 that supports it.
In a future PR I'll ensure `spread parameters after optional chaining` is transpiled.